### DISCUSSION TITLE
How can I make the scrolledtext uneditable

### DISCUSSION THREAD

**Discussion by EulbThgink** [israel-dryer/ttkbootstrap#409] [Q&A] [answered]
if I write code like this, the scrolledtext can not be edited from UI(user interface)   
===========================================  
        scrolled_text = ttkbootstrap.ScrolledText(
            master=self,
            highlightcolor=style.colors.primary,
            highlightbackground=style.colors.border,
            highlightthickness=1,
            font=("Courier", 13),
        )
        scrolled_text.bind("<Key>", lambda a: "break")
===========================================

However, If I write code like below, the scrolledtext can be edited from UI(user interface)
===========================================
from ttkbootstrap.scrolled import ScrolledText

         scrolled_text = ScrolledText(
            master=self,
            highlightcolor=style.colors.primary,
            highlightbackground=style.colors.border,
            highlightthickness=1,
            font=("Courier", 13),
            hbar=True
        )
        scrolled_text.bind("<Key>", lambda a: "break")
===========================================

My question is how can I make the scrolledtext uneditable by using 'from ttkbootstrap.scrolled import ScrolledText',
Thank you very much

**Comment by EulbThgink** [comment, marked as answer]
I think 
self.__scrolled_text.text.bind("<Key>", lambda a: "break")
will effect

**Comment by israel-dryer** [comment]
You can disable the text component by setting the state to "disabled".

```python
scrolled_text.text.configure(state="disabled")
```

**Reply by EulbThgink** [reply]
If I wrote like this 
`scrolled_text.text.configure(state="disabled")`
The process would not insert string anymore. So I think 
`scrolled_text.text.bind("",` lambda a: `"break")`
is what I want. I just do not want the user edit the scrolled text.
However, Thank you for your advice^_^
